setup.py

```diff
@@ -15,6 +15,7 @@
     'pytz>=2012c',
     'redis>=2.10',
     'requests>=2.2.0',
+    'msgpack==0.6.2',
```

Comment:
Please don't pin version numbers and use `>=` if possible.

(btw, on intelmq-dev we have 0.5.6 installed)